hdrop/Makefile

```diff
@@ -11,7 +11,7 @@ all: hdrop.1
 hdrop.1: hdrop.1.scd
 	scdoc < $< > $@
 
-install: hdrop.1 hdrop
```

Comment:
Why remove these? They were added in https://github.com/hyprwm/contrib/pull/80.

Reply:
Sorry, my bad.